lib/modules/manager/flux/extract.ts

```diff
@@ -197,6 +217,27 @@ function resolveResourceManifest(
         deps.push(dep);
         break;
       }
+      case 'OCIRepository': {
+        const dep: PackageDependency<FluxManagerData> = {
+          depName: resource.spec.url?.replace('oci://', ''),
+        };
+        if (resource.spec.ref?.digest) {
```

Comment:
if you don't like to support tag and digest as mentioned before you should log a warning, so users get a message that it's not supported. otherwise we'll get wrong bug reports about unexpected behavior.

Reply:
If only `digest` is set I have updated the currentValue to force being `latest` as that is what Flux assumes when only using `digest`.